server.js

```diff
@@ -68,10 +64,42 @@ app.get('/api/v1/players/:id', (request, response) => {
     });
 })
 
-app.get('/api/v1/players?:key=:value', (request, response) => {
-  database('players').where(request.params.key, request.params.value).select()
-    .then(() => {})
-})
+// app.get('/api/v1/players?:key=:value', (request, response) => {
+//   database('players').where(request.params.key, request.params.value).select()
+//     .then(() => {})
+// })
+
+app.post('/api/v1/teams', (request, response) => {
+  const { body } = request;
+
+  const parameters = ['team_name', 'head_coach', 'owner', 'most_recent_championship', 'defensive_rating', 'points_per_game'];
+
+  for (let requiredParameter of parameters) {
+    if(!body[requiredParameter]) {
+      return response
+        .status(422)
+        .json({ error: `You are missing ${requiredParameter} from the expected format`})
+    }
+  }
+
```

Comment:
This is solid and definitely makes the `for let` more readable with the array of keys being held outside the loop method.